src/Symfony/Component/HtmlSanitizer/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+6.1.0
+-----
+
+ * added the component as experimental
```

Comment:
```suggestion
 * Add the component as experimental
```

Reply:
Yes, this is how changelogs are written now.
https://github.com/symfony/symfony/blob/5.4.0/src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md#54